beacon-chain/slasher/chunks.go

```diff
@@ -199,10 +199,11 @@ func (m *MinSpanChunksSlice) CheckSlashable(
 				return &slashertypes.Slashing{
 					Kind:            slashertypes.SurroundingVote,
 					ValidatorIndex:  validatorIdx,
-					PrevSourceEpoch: existingAttRecord.IndexedAttestation.Data.Source.Epoch,
-					PrevTargetEpoch: existingAttRecord.IndexedAttestation.Data.Target.Epoch,
-					SourceEpoch:     sourceEpoch,
 					TargetEpoch:     targetEpoch,
+					PrevSigningRoot: existingAttRecord.SigningRoot,
```

Comment:
Not sure if we do, but let's ensure there are nil checks somewhere before this that would prevent any panics